testing/test_assertion.py

```diff
@@ -172,7 +172,11 @@ def check(values, value):
                 return check
             """,
             "mainwrapper.py": """\
-            import pytest, importlib_metadata
+            import pytest, sys
+            if sys.version_info >= (3, 8):
+                from importlib import metadata
+            else:
+                import importlib_metadata as metadata
```

Comment:
there is later `DummyDistInfo` class with class variable called `metadata`, however that doesn't seem to be the problem.